.github/workflows/ci_cd.yml

```diff
@@ -513,6 +514,7 @@ jobs:
     name: Nuxt Playwright VR tests
     if: needs.get-changes.outputs.frontend == 'true'
     runs-on: ubuntu-latest
+    timeout-minutes: 15
```

Comment:
This test takes ~12 min so 15 min might be cutting it close. Maybe a higher limit like 20 min might be more appropriate.

Reply:
12 is the max I've seen, with as low as 7. Could we keep this as-is and increase in future PRs which increase the duration?